src/components/GlobalHelper/GlobalHelper.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import {Popover, PopoverProps} from 'wix-ui-core/dist/src/baseComponents/Popover/Popover';
+import style from './GlobalHelper.st.css';
+import {withStylable} from 'wix-ui-core/withStylable';
+import {createComponentThatRendersItsChildren, ElementProps} from 'wix-ui-core/dist/src/utils';
+
+/**
+ * Adapts Popover API with Popover.Element, and Popover.Content into regular props
+ */
+export interface PopoverAdapterProps {
+  /** children to render that will be the target of the tooltip */
+  children: React.ReactNode;
+  /** the content to put inside the tooltip */
+  content: React.ReactNode;
+}
+
+export interface GlobalHelperOwnProps {
+}
+
+export type GlobalHelperProps = PopoverProps & PopoverAdapterProps & GlobalHelperOwnProps;
+
+export type GlobalHelperType = React.SFC<GlobalHelperProps>;
+
+const defaultProps = {
+};
+
+const getState: (p?: any, s?: any, c?: any) => StateMap = (p) => ({});
+
+const GlobalHelperBO = withStylable<PopoverProps, GlobalHelperOwnProps>(
+  Popover,
+  style,
+  getState,
+  defaultProps
+);
+
+export const GlobalHelper: GlobalHelperType = (props) => {
```

Comment:
might need to add a programmatic opening capability.
so the consumer could pass a variable to open it even without a user intention

Reply:
PopoverProps has a `shown` prop that does that.
When we'll make this an Uncontrolled component (with state), then we can add a programmatic `close()` method.